### ISSUE TITLE
Request: Checksums on download page

### ISSUE THREAD

**Issue by BEMZ01** [mapmapteam/mapmapteam.github.io#4]
Hi! Title says it all, would be nice to see the Checksums for downloads.

**Comment by aalex** [comment]
That should be ready to do. On the website? Can you provide a link to which page you are referring to?

Does GitHub do it automatically on the GitHub repository page?

**Comment by BEMZ01** [comment]
Yeah,  on Mapmapteam.github.io near the downloads section. Possibly just below ?

**Comment by BEMZ01** [comment]
Possibly here?
![image](https://user-images.githubusercontent.com/48926280/143610362-ec8eb337-fb2b-48f5-b91e-128213fb2f0a.png)
